README.md

```diff
@@ -77,6 +77,14 @@ sudo pip3 install safeeyes
 sudo gtk-update-icon-cache /usr/share/icons/hicolor
 ```
 
+### OpenSUSE Tumbleweed
+
+```bash
+sudo zypper refresh
+sudo zypper install cairo-devel python3-devel python3-pycairo
```

Comment:
Two points:
1. Devel packages are not required to run `safeeyes`, so I would remove those dependencies
2. The `python3-pycairo` package will be added as a runtime dependency soon ([request 838108](https://build.opensuse.org/request/show/838108)), so you won't need to explicitly state this either.

Reply:
I am not familiar with openSUSE. @Huy-Ngo once you've confimed, I will merge this PR. Thank you for your contribution.

Reply:
@cbcoutinho is correct. I'll change that now.